source/reference/json.txt

```diff
@@ -150,6 +154,17 @@ To learn more about alert configuration files, see
 :ref:`atlas-cli-alert-config-file`. For a sample file,
 see :ref:`example-alert-config-file`.
 
+Rolling Index Configuration File 
+--------------------------------
+
+To learn more about rolling index configuration files, see 
+:ref:`atlas-cli-rolling-index-config-file`. For example files, see:
+
+- `Unique indexes <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_2dspere_index.json>`__ 
+- `Sparse Index <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_sparse_index.json>`__ 
+- `Partial indexes <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_partial_index.json>`__ 
```

Comment:
Can we add these links as a "See Also" to the dedicated page as well so they don't get missed? Thank you!